doc/tutorials/mount.md

```diff
@@ -1,5 +1,6 @@
 # Mounting
 
+<!-- TODO [new_backend]: Outdated, only kept as reference until an updated version is written
```

Comment:
Is it already outdated or after @flo91's work? I understood that at least some mount functionality is working here?

Reply:
True, the commands should all (or almost all) work. I'll uncomment it again and add a warning that some of the explanation around the commands might be outdated.